src/Network/Network/NetworkManager/NetworkManagerSecurityAdminConfiguration/NewAzNetworkManagerSecurityAdminConfigurationCommand.cs

```diff
@@ -29,6 +30,10 @@ namespace Microsoft.Azure.Commands.Network
     [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetworkManagerSecurityAdminConfiguration", SupportsShouldProcess = true), OutputType(typeof(PSNetworkManagerSecurityAdminConfiguration))]
     public class NewAzNetworkManagerSecurityAdminConfigurationCommand : NetworkManagerSecurityAdminConfigurationBaseCmdlet
     {
+        // Breaking change 2022-05-01 release, plan to remove warning in 2022-07-01 release
+        public const string ApplyOnNetworkIntentPolicyBasedServiceChangeDesc = "Apply on network intent policy based service options are now restricted to 'None', 'All', and 'AllowRulesOnly'.";
+        [CmdletParameterBreakingChange("ApplyOnNetworkIntentPolicyBasedService", ChangeDescription = ApplyOnNetworkIntentPolicyBasedServiceChangeDesc)]
```

Comment:
`CmdletParameterBreakingChange` is used to notify user before we do breaking change. So you need to add this notification 2 releases before this major release.

Reply:
Please ping me on teams is you have any question

Reply:
Thanks for your help, I’ve removed the alerts